Doc/whatsnew/3.15.rst

```diff
@@ -354,6 +355,19 @@ tarfile
   (Contributed by Matt Prodani and Petr Viktorin in :gh:`112887`
   and :cve:`2025-4435`.)
 
+tkinter
+-------
+
+* The ``tkinter.Text.search`` method now supports two additional
```

Comment:
```suggestion
* The :meth:`!tkinter.Text.search` method now supports two additional
```
So it will be formatted with the same font and color as other methods, just without a link.